src/probnum/quad/bq_methods/stop_criteria/_stopping_criterion.py

```diff
@@ -0,0 +1,101 @@
+"""Stopping criteria for Bayesian quadrature."""
+
+from typing import Callable
+
+import numpy as np
+
+from probnum.quad.bq_methods.bq_state import BQState
+from probnum.randvars import Normal
+from probnum.type import FloatArgType, IntArgType
+
+
+class StoppingCriterion:
+    """Base class for a stopping criterion.
+
+    Parameters
+    ----------
+    stopping_criterion :
+        A function that determines whether a convergence criterion is reached.
+    """
+
+    def __init__(self, stopping_criterion: Callable):
+        self._stopping_criterion = stopping_criterion
+
+    def __call__(self, integral_belief: Normal, bq_state: BQState) -> bool:
+        """Evaluate the stopping criterion.
+
+        Parameters
+        ----------
+        integral_belief :
+            Current Gaussian belief about the integral.
+        bq_state:
+            State of the BQ loop.
+
+        Returns
+        -------
+        has_converged:
+            Boolean whether a stopping criterion has been reached
+        """
+        return self._stopping_criterion(bq_state)
+
+
+class IntegralVariance(StoppingCriterion):
+    """Stop once the integral variance is below some tolerance.
+
+    Parameters
+    ----------
+    var_tol:
+        Tolerance value of the variance.
+    """
+
+    def __init__(self, var_tol: FloatArgType = None):
+        self.var_tol = var_tol
+        super().__init__(stopping_criterion=self.__call__)
+
+    def __call__(self, integral_belief: Normal, bq_state: BQState) -> bool:
+        return integral_belief.var <= self.var_tol
+
+
+class RelativeError(StoppingCriterion):
```

Comment:
We also need a better name for this one (Although I can;t think of a good one right now.)

Reply:
`RelativeMeanChange`?

Reply:
went for this now